pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -732,3 +732,26 @@ SELECT * FROM pg_extension.geometry_columns
 
 statement error not yet implemented
 SELECT * FROM pg_extension.spatial_ref_sys ORDER BY srid ASC
+
+
+subtest st_srid
```

Comment:
Own subtest is required because the following errors are caused.
> pq: st_distance(): operation on mixed SRIDs forbidden: (Point, 4326) != (LineString, 0)